src/components/input-stuff.js

```diff
@@ -1,14 +1,98 @@
 import React from 'react';
+import {connect} from 'react-redux';
 
+import {addBrew} from '../database/brew';
+
+var testEquip = ["Chemex", "Clever", "French Press"];
 
 const FancyInput = (props) => {
   return (
-  	<label className="input-label"> {props.label} 
-  	<input name={props.name} type="text" />
-  	</label> )
+ 	    <label className="input-label"> {props.label} 
+      <input name={props.name} type="text" />
+      </label> 
+  )
+}
+
+const FancyDropdown = (props) => {
+  return (
+    <div>
+      <label className="input-label"> {props.label}
+      <select name={props.name}>
+        {props.options.map(equip => <option>{equip}</option>)}
+      </select>
+      </label>
+    </div>)
 }
 
-// const FancyDropdown = (props)
 
+const FancyButton = (props) => {
+  return (
+    <button name={props.name} type="submit" onClick={evt => console.log("")}>
+      Submit
+    </button>
+    )
+}
+
+///
+
+const FancyForm = (props) => {
+
+  // So I started out trying to make this generalizable, a FancyForm that can be 
+  // used in other places. But now I've noticed that I'm making it very specific to
+  /// just adding a brew. Should I be handling this submit in BrewInput instead?
+
+  function handleSubmit(event) {
+
+    console.log(props.userid);
+    
+    event.preventDefault();
+    /// still missing: coffeeRef, rating, waterBrewAmount, waterSoakAmount
+
+    const newBrew = {};
+    const timeNow = new Date();
+
+    newBrew['brewMethod'] = event.target.brewMethod.value;
+    newBrew['coffeeAmount'] = event.target.amount.value;
+    newBrew['brewDate'] = event.target.brewDate.value;
+    newBrew['grindSetting'] = event.target.grindSetting.value;
+    newBrew['name'] = event.target.name.value;
+    newBrew['brewTime'] = event.target.time.value;
+    newBrew['dateCreated'] = timeNow.getTime();
+    newBrew['notes'] = event.target.notes;
```

Comment:
since this is a pretty low-key object, you can create future ones like it like this:
```javascript
const newBrew = {
  brewMethod = event.target.brewMethod.value,
  brewTime = event.target.time.value,
  ...
};
```
just a little less typing